opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/container-authorized-members-test.ts

```diff
@@ -0,0 +1,666 @@
+      query settings {
+        settings {
+          id
+          platform_organization {
+            id
+            name
+          }
+        }
+      }
+    `;
+    const queryResult = await adminQuery({ query: SETTINGS_READ_QUERY, variables: {} });
+    settingsInternalId = queryResult.data?.settings?.id;
+
+    // Set EE
+    const EEqueryResult = await adminQuery({
+      query: PLATFORM_ORGANIZATION_QUERY,
+      variables: {
+        id: settingsInternalId,
+        input: [
+          { key: 'enterprise_edition', value: new Date().getTime() },
+        ]
+      }
+    });
+    expect(EEqueryResult).not.toBeNull();
+    expect(EEqueryResult?.data?.settingsEdit.fieldPatch.enterprise_edition).not.toBeUndefined();
+  });
+  it('should share Case Incident Response with Organization', async () => {
+    // Get organization id
+    organizationId = await getOrganizationIdByName(PLATFORM_ORGANIZATION.name);
+    const organizationSharingQueryResult = await adminQuery({
+      query: ORGANIZATION_SHARING_QUERY,
+      variables: { id: caseIrId, organizationId }
+    });
+    expect(organizationSharingQueryResult).not.toBeNull();
+    expect(organizationSharingQueryResult?.data?.stixCoreObjectEdit.restrictionOrganizationAdd).not.toBeNull();
+    expect(organizationSharingQueryResult?.data?.stixCoreObjectEdit.restrictionOrganizationAdd.objectOrganization[0].name).toEqual(PLATFORM_ORGANIZATION.name);
+  });
+  it('should Editor user from different organization access Case Incident Response', async () => {
+    const caseIRQueryResult = await editorQuery({ query: READ_QUERY, variables: { id: caseIrId } });
+    expect(caseIRQueryResult).not.toBeNull();
+    expect(caseIRQueryResult?.data?.caseIncident).not.toBeUndefined();
+    expect(caseIRQueryResult?.data?.caseIncident.id).toEqual(caseIrId);
+  });
+  it('should Security user from shared organization access Case Incident Response', async () => {
+    const caseIRQueryResult = await securityQuery({ query: READ_QUERY, variables: { id: caseIrId } });
+    expect(caseIRQueryResult).not.toBeNull();
+    expect(caseIRQueryResult?.data?.caseIncident).not.toBeUndefined();
+    expect(caseIRQueryResult?.data?.caseIncident.id).toEqual(caseIrId);
+  });
+  it('should Case Incident Response deleted', async () => {
+    // Delete the case
+    await adminQuery({
+      query: DELETE_QUERY,
+      variables: { id: caseIrId },
+    });
+    // Verify is no longer found
+    const queryResult = await adminQuery({ query: READ_QUERY, variables: { id: caseIrId } });
+    expect(queryResult).not.toBeNull();
+    expect(queryResult?.data?.caseIncident).toBeNull();
+  });
+  it('should EE deactivated', async () => {
+    const platformOrganization = await adminQuery({
+      query: PLATFORM_ORGANIZATION_QUERY,
+      variables: { id: settingsInternalId,
+        input: [
+          { key: 'enterprise_edition', value: [] },
+        ] }
+    });
+    expect(platformOrganization).not.toBeNull();
```

Comment:
non blocking: if you use queryAsAdminWithSuccess it avoid to check the not null after.